cub/test/c2h/generators.cuh

```diff
@@ -110,5 +145,34 @@ template <typename T>
 thrust::device_vector<T>
 gen_uniform_offsets(seed_t seed, T total_elements, T min_segment_size, T max_segment_size);
 
-} // c2h
+/**
+ * @brief Generates key-segment ranges from an offsets-array like the one given by
+ * `gen_uniform_offset`.
+ */
+template <typename key_t, typename offset_t>
```

Comment:
```suggestion
template <typename KeyT, typename OffsetT>
```

Reply:
Changed template parameters to follow `CamelCaseT` across the board.